substrate/client/cli/src/params/node_key_params.rs

```diff
@@ -79,22 +80,47 @@ pub struct NodeKeyParams {
 	/// the chosen type.
 	#[arg(long, value_name = "FILE")]
 	pub node_key_file: Option<PathBuf>,
+
+	/// Forces key generation if node-key-file file does not exist.
+	///
+	/// This is an unsafe feature for producation networks because the previous
+	/// node address would still live in the DHT for 36h and the node would suffer
+	/// from bad connectivity to other nodes untill the old record expires.
+	/// For minimal node downtime if no custom `node-key-file` argument is provided
+	/// the network-key is usually persisted accross nodes restarts,
+	/// in the `network` folder from directory provided in `--base-path`
+	///
+	/// This is safe to be used the first time you run your node or if you already
+	/// lost your node keys or if you don't care about the above problem.
```

Comment:
I wouldn't add a comment that it's safe, even under specific circumstances, — people might use it and forget they passed this argument.

Reply:
It does says you need to remove it, I just wanted people to be able to bootstrap the first time they run a node without much inconvenience.

Reply:
IMO it's better to recommend a user to use an explicit command, than something that can be misused.

Reply:
I ended-up keeping this as well.

Reply:
It doesn't seem right to me to start a phrase with words `This is safe ...` for an argument that we explicitly marked as `--unsafe-...`. If we want to discourage people using it, let's be consistent. Especially as we have a safe one-off alternative. First of all, the purpose of this PR is the opposite to running the node with this argument.

Reply:
Will update the comment you are right.

Reply:
Should be better now.

Reply:
Looks good, thanks!